### ISSUE TITLE
Clean Upgrade Produces "Failed to Load Resource" and missing FontAwesome?

### ISSUE THREAD

**Issue by chewy-chewbacca** [projectsend/projectsend#852]
In troubleshooting my [email issue](https://github.com/projectsend/projectsend/issues/848), I decided to remove our entire ProjectSend installation as I saw files still existing from r1070 on our server that didn't exist in the r1295 zip. I literally removed everything, replaced all files and copied in our sys,config.php file. Unfortunately that didn't fix anything with email and broke more stuff.

Now it seems we are missing the graphs on the homepage and I believe fontawesome is broken.

https://drive.google.com/open?id=1pwT1rhC_X8ZoJ4C7ldWmvFcoEvqLi9LS

Wondering if we should just revert back to r1070 as that worked perfectly and wait for this version to be more mature.

**Comment by chewy-chewbacca** [comment]
Tried again. Full delete and reinstall, same previously upgraded database, which resulted in the same failed to load resource on graphs, preview broken with same, and fontawesome seems to be missing... looks the same as the screenshot prior. There must be a dependency to some files from a previous version I guess.

**Comment by chewy-chewbacca** [comment]
For kicks did a clean install on a separate Plesk which worked fine. Bizarre. I gave up and restored r1070 for my client, for now.

**Comment by ignacionelson** [comment]
I see. Seems like ajax is not returning a json response, that's why you see the Failed to load resource message.
You need to enable debug mode in includes/app.php, and then open the browser console, go to the Network tab, reload the page and check the Response of any of the widget's ajax calls. The error should be there

**Comment by ignacionelson** [comment]
Which php version are you using?

**Comment by chewy-chewbacca** [comment]
I had to roll the production server back, so I can't test further. It's okay for now, it's got to be something I'm doing wrong. Thanks for the help and the project!

**Comment by chewy-chewbacca** [comment]
Ignacio, just FYI I restored the database in question to the new install and it all worked fine. Again, I don't think this is a problem for you, but something with the hosted environment, etc. Don't waste any more of your time on this one.

**Comment by chewy-chewbacca** [comment]
Hey Ignacio - I finally got back to this and the issue at hand was that we are piggybacking this off a website. The website used www and we were using no-WWW in the portal creating a cross-site scripting security issue. Once I updated the portal to match the website everything just worked. This time for the upgrade I renamed our PJ folder to old, uploaded the new version and renamed it to our original PJ location. Ran the database upgrade and it worked like a charm. 

Not sure if you want to amend your install instructions, because simply overwriting in FileZilla leaves all the old stuff from previous versions behind in my experience and possibly security holes. My clean install worked great.
